<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>521</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_back">
       <property name="text">
        <string>push_back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_front">
       <property name="text">
        <string>push_front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pop_back">
       <property name="text">
        <string>pop_back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pop_front">
       <property name="text">
        <string>pop_front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="get_value">
       <property name="text">
        <string>get value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="is_empty">
       <property name="text">
        <string>empty </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>209</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
